pyramid_restful/services/restful.py

```diff
@@ -1,63 +0,0 @@
-import re
-
-from .abstract import AbstractService
-from pydoc import render_doc
-from pyramid.httpexceptions import HTTPBadRequest
-
-
-# noinspection PyMethodOverriding
-class RestfulService(AbstractService):
```

Comment:
not needed...this was only used in pyramid_orb, and I've just put the base logic in there